ambry-tools/src/main/java/com.github.ambry/store/DumpDataHelper.java

```diff
@@ -140,44 +144,52 @@ public void dumpLog(File file, long startOffset, long endOffset, ArrayList<Strin
       long tempCurrentOffset = currentOffset;
 
       try {
-        BlobRecordInfo blobRecordInfo = readSingleRecordFromLog(randomAccessFile, currentOffset);
+        LogBlobRecordInfo logBlobRecordInfo = readSingleRecordFromLog(randomAccessFile, currentOffset);
```

Comment:
where is `LogBlobRecordInfo`? Doesn't seem to be on this PR or on the master branch.